app/transformations/UserAnswersToETMP/API1830.scala

```diff
@@ -213,11 +243,101 @@ object API1830 extends Transformer {
     case _ => fail[JsString]
   }
 
+  private val readsMemberHoldProtection: Reads[JsString] = (__ \ Symbol("validProtection")).json.pick.flatMap {
+    case JsBoolean(value) => Reads.pure(toYesNo(JsBoolean(value)))
+    case _ => fail[JsString]
+  }
+
+  private val readsAvailableLumpSumExceeded: Reads[JsString] = (__ \ Symbol("overAllowance")).json.pick.flatMap {
+    case JsBoolean(value) => Reads.pure(toYesNo(JsBoolean(value)))
+    case _ => fail[JsString]
+  }
+
+  private val readsAvailableLumpSumDBAExceeded: Reads[JsString] = (__ \ Symbol("overAllowanceAndDeathBenefit")).json.pick.flatMap {
+    case JsBoolean(value) => {
+      Reads.pure(toYesNo(JsBoolean(value)))
+    }
+    case _ => {
+      fail[JsString]
+    }
+  }
+
+  private val readsSchemeSpecificLumpSum: Reads[JsString] = (__ \ Symbol("typeOfProtectionGroup1")).json.pick.flatMap {
+    case JsArray(value) => {
```

Comment:
I think we can simplify these cases so that we don't need to repeat with fail cases:

`case JsArray(value) if value.contains(JsString("schemeSpecific")) => ...`
`case _ => ...`